libraries/ui/src/ValueCard.tsx

```diff
@@ -1,20 +1,32 @@
+import clsx from 'clsx';
 import React from 'react';
 
 interface ValueCardProps {
   icon: string;
-  title?: string;
   description: string;
+  title?: string;
+  dark?: boolean;
+  className?: string;
 }
 
-export const ValueCard: React.FC<ValueCardProps> = ({ icon, title, description }) => {
+export const ValueCard: React.FC<ValueCardProps> = ({
+  icon, title, description, dark = false, className,
+}) => {
   return (
-    <div className="value-card w-full flex flex-col gap-6 py-6">
-      <div className="value-card__icon-wrapper size-[50px] flex items-center justify-center">
-        <img
-          src={icon}
-          alt={title}
-          className="value-card__icon"
-        />
+    <div
+      className={clsx(
+        'value-card size-full flex flex-col gap-6 py-6 rounded-radius-md',
+        dark ? 'bg-color-canvas-dark [&_*]:text-white px-6' : 'bg-color-canvas',
+        className,
+      )}
+    >
+      <div
+        className={clsx(
+          'value-card__icon-wrapper flex items-center',
+          dark ? 'flex-1 ml-[10%]' : 'justify-center size-[50px]',
+        )}
+      >
+        <img src={icon} alt={title} className="value-card__icon" />
```

Comment:
This changing of the image classes based on `dark` is a bit overcomplicated, but we can remove this `dark` option entirely soon (#260) and get rid of this